packages/core/src/browser/fetchObservable.spec.ts

```diff
@@ -33,6 +33,10 @@ describe('fetch proxy', () => {
   })
 
   afterEach(() => {
+    if (isIE()) {
+      return
+    }
+
```

Comment:
**💬 suggestion:** ‏Instead of duplicating the condition in afterEach (which is error-prone, as it can be forgotten), you could leverage `registerCleanupTask` in `beforeEach` and remove the `afterEach` usage. For example, in this test, we could write:

```ts
 beforeEach(() => {
    if (isIE()) {
      pending('no fetch support')
    }
    fetchStubManager = stubFetch()
    originalFetchStub = window.fetch
    requests = []
    requestsTrackingSubscription = initFetchObservable().subscribe((context) => {
      if (context.state === 'resolve') {
        requests.push(context)
      }
    })
    fetchStub = window.fetch as FetchStub
    registerCleanupTask(() => {
      requestsTrackingSubscription.unsubscribe()
      contextEditionSubscription?.unsubscribe()
      fetchStubManager.reset()
    })
  })
```

This could be a step toward https://datadoghq.atlassian.net/browse/RUM-5101